myia/operations/op_full.py

```diff
@@ -0,0 +1,45 @@
+"""Implementation of operation for numpy.full."""
+
+import numpy as np
+
+from ..lib import (
+    abstract_array,
+    core,
+    distribute,
+    myia_to_array,
+    scalar_cast,
+    to_scalar_type,
+    typeof,
+)
+
+
+def pyimpl_full(shape, fill_value, dtype):
+    """Python implementation for operation full."""
+    return np.full(shape, fill_value, dtype)
+
+
+@core
+def full(shape, fill_value, dtype):
```

Comment:
You should be able to have default arguments here.

```suggestion
def full(shape, fill_value, dtype=None):
```

Reply:
Done!